### PR TITLE
add internal property warning

### PR DESCRIPTION
**Pull request by Deflaimun**
Resolves https://github.com/redpanda-data/documentation-private/issues/2373
Fixes: DOC-198

### CODE REVIEW

File: modules/manage/partials/internal-use-property.adoc [hunk 1]
```diff
@@ -0,0 +1 @@
+WARNING: This property is exclusive for internal use. Please do not modify it unless specifically instructed by Redpanda Support.
```
Comment (kbatuigas) [inline modules/manage/partials/internal-use-property.adoc]: I'd phrase it along the lines of:
`WARNING: This property is for Redpanda internal use only. Do not modify it unless specifically instructed by Redpanda Support.`

Comment (Deflaimun) [inline modules/manage/partials/internal-use-property.adoc]:
```suggestion
WARNING: This property is for Redpanda internal use only. Do not modify it unless specifically instructed by Redpanda Support.
```

File: modules/manage/partials/internal-use-property.adoc [hunk 2]
```diff
@@ -0,0 +1 @@
+WARNING: This property is for Redpanda internal use only. Do not modify it unless specifically instructed by Redpanda Support.
```
Comment (wzzzrd86) [inline modules/manage/partials/internal-use-property.adoc]: Can we link to [https://support.redpanda.com/hc/en-us](https://support.redpanda.com/hc/en-us)

Comment (Deflaimun) [reply modules/manage/partials/internal-use-property.adoc]: Yes! Sounds good to me

Comment (Deflaimun) [inline modules/manage/partials/internal-use-property.adoc]:
```suggestion
WARNING: This property is for Redpanda internal use only. Do not modify it unless specifically instructed by https://support.redpanda.com/hc/en-us[Redpanda Support^].
```

File: modules/manage/partials/internal-use-property.adoc [hunk 3]
```diff
@@ -0,0 +1 @@
+WARNING: This property is for Redpanda internal use only. Do not modify it unless specifically instructed by https://support.redpanda.com/hc/en-us[Redpanda Support^].
```
Comment (Feediver1) [inline modules/manage/partials/internal-use-property.adoc]: So does this appear once at the top of the topic? If so, then:

WARNING: _This property is for Redpanda internal use only_. Do not use or modify any property here unless specifically instructed to do so by https://support.redpanda.com/hc/en-us[Redpanda Support^]. Using any property here without explicit guidance from Redpanda support could result in data loss.

Comment (Deflaimun) [reply modules/manage/partials/internal-use-property.adoc]: No. The idea here is to inject this admonition into every property that is meant for internal-only usage. 

Do you want to create a separate section for those properties?

Comment (Deflaimun) [reply modules/manage/partials/internal-use-property.adoc]: Going with individual admonitions for properties after a decision in a Zoom Call.

Comment (Deflaimun) [inline modules/manage/partials/internal-use-property.adoc]:
```suggestion
WARNING: This property is for Redpanda internal use only. Do not use or modify this property unless specifically instructed to do so by [https://support.redpanda.com/hc/en-us[Redpanda](https://support.redpanda.com/hc/en-us%5BRedpanda) Support^]. Using this property without explicit guidance from Redpanda Support could result in data loss.
```

**Review by Feediver1 — APPROVED** [review]

**Pull request merged**